pkg/result/processors/path_prefixer_test.go

```diff
@@ -0,0 +1,37 @@
+package processors
+
+import (
+	"go/token"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/golangci/golangci-lint/pkg/result"
+)
+
+func TestPathPrefixer_Process(t *testing.T) {
+	paths := func(ps ...string) (issues []result.Issue) {
+		for _, p := range ps {
+			issues = append(issues, result.Issue{Pos: token.Position{Filename: p}})
+		}
+		return
+	}
+	for _, tt := range []struct {
+		name, prefix string
+		issues, want []result.Issue
+	}{
+		{"empty prefix", "", paths("some/path", "cool"), paths("some/path", "cool")},
+		{"prefix", "ok", paths("some/path", "cool"), paths("ok/some/path", "ok/cool")},
+		{"prefix slashed", "ok/", paths("some/path", "cool"), paths("ok/some/path", "ok/cool")},
+	} {
+		t.Run(tt.name, func(t *testing.T) {
+			r := require.New(t)
+
+			p := NewPathPrefixer(tt.prefix)  //nolint:scopelint
```

Comment:
My goland auto generated tabular test is exactly the same here, seems like this was discussed here https://github.com/kyoh86/scopelint/issues/4

We can just skip this scopelint here.

Reply:
Yeah; the closure is only a problem if it's executed after the iteration of the loop, which it's not -- but it'd be very tough for the scopelint tool to recognize that closures are executing more or less immediately here.

As it is, people passing closures out of loops and messing up the bindings is such a common toe stub, it makes sense to me to have a CI tool that will yell, even if it is with a lot of false positives.